packages/nhost-js/tests/nhost.test.ts

```diff
@@ -1,5 +1,5 @@
 import { vi } from 'vitest'
```

Comment:
@plmercereau is there a reason why you are moving this outside of `clients`?

Reply:
yes,
everything in `tests` is related to unit tests. They don't run tests against a real backend such as the ones put in the `e2e` folder. In our unit tests, we are checking the instantiation of the backend url(s) of Nhost client, but not its effective functioning.
The `nhost.graphql` client is tested against a Nhost CLI backend in the `e2e` part, and parts (auth & storage) are tested in their respective packages.

In our CI, all the unit tests are run together using turborepo with `pnpm run test`, whereas `e2e` each require the CLI to start, with a possible custom configuration defined in `package/x/nhost`. They cannot be run in parallel in the same machine (In GitHub actions, they are run in parallel on separate machines).